drivers/sensor/wsen_hids/Kconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig HIDS
```

Comment:
IMO this should be WSEN_HIDS. "HIDS" is too easy to mistake for [human interface devices](https://en.wikipedia.org/wiki/Human_interface_device).

Reply:
True. I had first named all the new WE drivers/identifiers similar to the existing ITDS driver, but using the full name including the "WSEN" prefix makes sense.